third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -968,26 +971,26 @@ void ParagraphTxt::Layout(double width) {
             // The placeholder run's layout should yield one glyph representing
             // the object replacement character.  Replace its width with the
             // placeholder's width.
-            FML_DCHECK(layout.nGlyphs() == 1);
             glyph_advance = run.placeholder_run()->width;
+            glyph_positions.emplace_back(
```

Comment:
Would it be possible to insert the placeholder's `codepoint_length` into `grapheme_code_unit_widths` and then use the current logic for populating `glyph_positions`?

Reply:
I think the goal is to have all of the obj replacement chars to be "stacked" on top of each other with the same coordinates. This way, indexing any of the chars in the range will result in the same results.

Reply:
What is the difference between that and a single glyph that consumes `codepoint_length` characters?